### DISCUSSION TITLE
Requesting to have bullets removed from Task Lists

### DISCUSSION THREAD

**Discussion by marlonjames71** [notaapp/talk#360] [Feature Requests]
## From This:
<img width="298" alt="CleanShot 2023-02-02 at 02 54 03@2x" src="https://user-images.githubusercontent.com/48145861/216264433-449d1812-16d4-4242-b732-c2893af3627b.png">

## To This:
<img width="346" alt="CleanShot 2023-02-02 at 02 54 12@2x" src="https://user-images.githubusercontent.com/48145861/216264536-9260caff-8500-401e-aa6b-828fb655010b.png">

---

It's noise and doesn't really serve a purpose in my opinion. Looks way cleaner with just the checkboxes and is less things to select/highlight.

Thanks!

**Comment by erusev** [comment]
It does look cleaner, but we'd have to figure out how folding would work, as now we use bullets as buttons.

**Reply by marlonjames71** [reply]
I would imagine that a chevron icon/button would appear in the gutter upon hovering over the parent list item.

**Reply by marlonjames71** [reply]
...that would be if a clickable button is even needed. There's already a great keyboard shortcut, and it's in the command palette as well.

**Reply by erusev** [reply]
Yes, I think we can find a nice way to do it, thanks for the suggestions.

**Reply by marlonjames71** [reply]
Sure.

**Comment by luvs** [comment]
I like bullets with task lists, perfect if it's will be optional

**Reply by marlonjames71** [reply]
Too noisy IMO
